app/src/main/java/com/openclassrooms/realestatemanager/model/Estate.kt

```diff
@@ -21,6 +24,8 @@ data class Estate(
     val zipCode: String,
     val city: String,
     val country: String,
+    var latitude: Double?,
+    var longitude: Double?,
```

Comment:
Il est préférable de manipuler des objets immuables, quit à utiliser la méthode `copy()` sur les data classes. 

Je te propose une alternative plus bas, pour avoir ces valeurs à la construction de l'objet.

```suggestion
    val latitude: Double?,
    val longitude: Double?,
```

Reply:
ok

Reply:
Je savais que tu allais me parler de ça :)

Reply:
https://github.com/Tgarfr/Project-9-RealEstateManager/pull/10/commits/a1361f179b13ed7e4a94329d60e32d28e458e19d